### PR TITLE
Cpp math hypot

### PR DESCRIPTION
**Pull request by Sparsh47**
### Checklist
- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by Sparsh47** [comment]
I have made all the changes asked  by you and also removed the two commits.

File: content/cpp/concepts/math-functions/terms/hypot/hypot.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+Title: 'hypot()'
+Description: 'Returns the square root of sum of square of arguments passed.'
```
Comment (SSwiniarski) [inline content/cpp/concepts/math-functions/terms/hypot/hypot.md:3]:
```suggestion
Description: 'Returns the square root of sum of squares of arguments passed.'
```

File: content/cpp/concepts/math-functions/terms/hypot/hypot.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+The **hypot()** function returns the square root of sum of square of arguments passed.
+
+## Syntax
+
+```cpp
+ hypot(float x, float y)
+```
+
+It finds the hypotenuse, hypotenuse is the longest side of a right angled triangle.
```
Comment (SSwiniarski) [inline content/cpp/concepts/math-functions/terms/hypot/hypot.md:23]:
```suggestion
The `hypot()` function finds the hypotenuse, the longest side of a right angled triangle. The parameters `x` and `y` are the lengths of the other two sides.
```

File: content/cpp/concepts/math-functions/terms/hypot/hypot.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+## Example
+
+The following example uses the `hypot()` function to find the hypotanuse of a right angled triangle with base and perpendicular as `9` & `10`:
```
Comment (SSwiniarski) [inline content/cpp/concepts/math-functions/terms/hypot/hypot.md:27]:
```suggestion
The following example uses the `hypot()` function to find the hypotenuse of a right triangle with base and perpendicular sides of `9` & `10`:
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Sparsh47, I left a couple more review comments.

**Comment by Sparsh47** [comment]
I have commited all the changes

File: content/cpp/concepts/math-functions/terms/hypot/hypot.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+```cpp
+ hypot(float x, float y)
+```
+
+The `hypot()` function finds the hypotenuse, the longest side of a right angled triangle. The parameters `x` and `y` are the lengths of the other two sides.
+
+## Example
+
+The following example uses the `hypot()` function to find the hypotenuse of a right triangle with base and perpendicular sides of `9` & `10`:
```
Comment (SSwiniarski) [inline content/cpp/concepts/math-functions/terms/hypot/hypot.md:27]:
```suggestion
The following example uses the `hypot()` function to find the hypotenuse of a right triangle with base and height of `9` & `10`:
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
One more change.

**Comment by Sparsh47** [comment]
I have commited this too.

**Review by SSwiniarski — APPROVED** [review]
@Sparsh47 This looks ready for a second review.

File: content/cpp/concepts/math-functions/terms/hypot/hypot.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **hypot()** function returns the square root of sum of square of arguments passed.
+
+## Syntax
+
+```cpp
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/hypot/hypot.md:19]:
```suggestion
```pseudo
```

File: content/cpp/concepts/math-functions/terms/hypot/hypot.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+using namespace std;
+int main()
+{
+    double x = 9, y = 10, res;
+    res = hypot(x, y);
+  
+    cout << res << endl;
+    return 0;
+    This results in the following output: 13.4536
+}
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/hypot/hypot.md:41]:
```suggestion

int main() {
  double base = 9, height = 10, res;
  res = hypot(base, height);
  
  cout << res << endl;
  return 0;
}
```

The last part didn't seem to fit right in the \```cpp code block.

File: content/cpp/concepts/math-functions/terms/hypot/hypot.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+```
+
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/hypot/hypot.md:43]:
```suggestion

This results in the following output:

```

Comment (Dusch4593) [reply content/cpp/concepts/math-functions/terms/hypot/hypot.md:43]: Then I'd suggest adding a new \```shell snippet that looks like this:

```
13.4536
```

File: content/cpp/concepts/math-functions/terms/hypot/hypot.md [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+## Codebyte Example
+
+The following example is runnable and returns the square root of sum of squares of `4.525252` & `5.767676`:
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/hypot/hypot.md:46]:
```suggestion
The following example is runnable and returns the square root of sum of squares of `4.525252` and `5.767676`:
```

File: content/cpp/concepts/math-functions/terms/hypot/hypot.md [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+int main()
+{
+    long double a, b, result;
+    a = 4.525252;
+    b = 5.767676;
+  
+    result = hypot(a, b);
+    cout << result;
+    return 0;
+}
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/hypot/hypot.md:61]:
```suggestion

int main() {
  long double a, b, result;
  a = 4.525252;
  b = 5.767676;
  
  result = hypot(a, b);
  cout << result;
  return 0;
}
```

File: content/cpp/concepts/math-functions/terms/hypot/hypot.md [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+  - 'Methods'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **hypot()** function returns the square root of sum of square of arguments passed.
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/hypot/hypot.md:15]:
```suggestion
The **hypot()** function returns the value of the longest side of a right-angled triangle.
```

File: content/cpp/concepts/math-functions/terms/hypot/hypot.md [hunk 11]
```diff
@@ -0,0 +1,62 @@
...
+
+## Syntax
+
+```cpp
+ hypot(float x, float y)
+```
+
+The `hypot()` function finds the hypotenuse, the longest side of a right angled triangle. The parameters `x` and `y` are the lengths of the other two sides.
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/hypot/hypot.md:23]:
```suggestion
The `hypot()` function finds the hypotenuse, the longest side of a right-angled triangle. The parameters `x` and `y` are the floating-point lengths of the other two sides.
```

File: content/cpp/concepts/math-functions/terms/hypot/hypot.md [hunk 12]
```diff
@@ -0,0 +1,62 @@
...
+
+## Example
+
+The following example uses the `hypot()` function to find the hypotenuse of a right triangle with base and height of `9` & `10`:
```
Comment (Dusch4593) [inline content/cpp/concepts/math-functions/terms/hypot/hypot.md:27]:
```suggestion
The following example uses the `hypot()` function to find the hypotenuse of a right triangle with `base` and `height` values:
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@Sparsh47 Finished with my review and left you some comments/suggestions.

**Comment by Dusch4593** [comment]
@Sparsh47 did you have any questions or need any help with implementing the latest round of feedback? As a reminder, today is the last day to merge your PRs and have them count toward Hacktoberfest. 👀

**Comment by SSwiniarski** [comment]
@Sparsh47 because it's been over two weeks of inactivity on this PR, I went and committed all of @Dusch4593's suggestions in the interest of getting this merged.

@Dusch4593, does anything else need to be done before approval?

File: content/cpp/concepts/math-functions/terms/hypot/hypot.md [hunk 13]
```diff
@@ -0,0 +1,67 @@
...
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+  - 'Methods'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **hypot()** function returns the value of the longest side of a right-angled triangle.
```
Comment (yangc95) [inline content/cpp/concepts/math-functions/terms/hypot/hypot.md:15]:
```suggestion
The **`hypot()`** function returns the value of the longest side of a right-angled triangle.
```

File: content/cpp/concepts/math-functions/terms/hypot/hypot.md [hunk 14]
```diff
@@ -0,0 +1,67 @@
...
+
+## Syntax
+
+```pseudo
+ hypot(float x, float y)
```
Comment (yangc95) [inline content/cpp/concepts/math-functions/terms/hypot/hypot.md:20]:
```suggestion
hypot(float x, float y)
```

File: content/cpp/concepts/math-functions/terms/hypot/hypot.md [hunk 15]
```diff
@@ -0,0 +1,67 @@
...
+
+## Example
+
+The following example uses the `hypot()` function to find the hypotenuse of a right triangle with `base` and `height` values:
+
+```cpp
+#include <cmath>
+#include <iostream>
+using namespace std;
+int main()
```
Comment (yangc95) [inline content/cpp/concepts/math-functions/terms/hypot/hypot.md:33]:
```suggestion

int main()
```

File: content/cpp/concepts/math-functions/terms/hypot/hypot.md [hunk 16]
```diff
@@ -0,0 +1,67 @@
...
+## Codebyte Example
+
+The following example is runnable and returns the square root of sum of squares of `4.525252` and `5.767676`:
+
+```codebyte/cpp
+#include <cmath>
+#include <iostream>
+using namespace std;
+
+int main() {
```
Comment (yangc95) [inline content/cpp/concepts/math-functions/terms/hypot/hypot.md:58]:
```suggestion
int main()
{
```

File: content/cpp/concepts/math-functions/terms/hypot/hypot.md [hunk 17]
```diff
@@ -0,0 +1,67 @@
...
+#include <cmath>
+#include <iostream>
+using namespace std;
+int main()
+{
+    double x = 9, y = 10, res;
+    res = hypot(x, y);
+  
+    cout << res << endl;
+    return 0;
```
Comment (yangc95) [inline content/cpp/concepts/math-functions/terms/hypot/hypot.md:39]:
```suggestion
  double x = 9, y = 10, res;
  res = hypot(x, y);
  
  cout << res << endl;
  return 0;
```

**Review by yangc95 — COMMENTED** [review]
@Sparsh47 I had some additional comments.

**Comment by SSwiniarski** [comment]
@yangc95, since I'd already committed @Dusch4593's comments for inactivity, in the interest of getting this merged I've also committed yours.

@yangc95, @Dusch4593, anything more we need before merging this?

**Review by Dusch4593 — APPROVED** [review]
cc: @Sparsh47 @yangc95 @SSwiniarski 

This LGTM 👍🏻

I made some edits in 601e5aa9f087c386b67ca37297a717cd2af00b03 that I'd like to note:

- Edited lines # 3 and # 15 to more closely resemble each other.
- Removed the data type markers from the \```shell snippet in the "Syntax" section.
  - These parameters can also be `double` or `long double`.
- Added a list of additional edge cases for the `x` and `y` parameters.
  - Included a link to the existing entry on the `fabs()` function.
- Light edit on lines # 34 and # 58.
- Adjusted syntax for `int main()` in the \```cpp snippet.

**Comment by SSwiniarski** [comment]
Congrats and thank you so much for contributing to Codecademy Docs, @Sparsh47! 🎊 🎉 🥳

The entry should now be live: https://www.codecademy.com/resources/docs/cpp/math-functions/hypot

Currently, your Codecademy profile is listed anonymously on the entry page. To make it visible, you'll need to add your GitHub username to your Codecademy profile, log out of Codecademy, and log back in through your GitHub account:

**Pull request merged**